keel-api/src/main/proto/Asset.proto

```diff
@@ -14,6 +14,18 @@ message Asset {
   google.protobuf.Any spec = 4;
 }
 
+message PartialAsset {
+  AssetId id = 1;
```

Comment:
why does this have its own id?

Reply:
It needs to be addressable. Partial assets will be saved / deleted independently from assets.

Reply:
Ok, I guess that makes sense.